modules/installation-vsphere-machines.adoc

```diff
@@ -156,7 +156,7 @@ $ export IPCFG="ip=192.168.100.101::192.168.100.254:255.255.255.0:::none nameser
 $ govc vm.change -vm "<vm_name>" -e "guestinfo.afterburn.initrd.network-kargs=${IPCFG}"
 ----
 +
-*** Optional: In the event of cluster performance issues, from the *Latency Sensitivity* list, select *High*.
+*** Optional: In the event of cluster performance issues, from the *Latency Sensitivity* list, select *High*. You also need to set the VM's CPU and memory reservation to its configured memory size.
```

Comment:
How about change to "You also need to set VM's CPU reservation and memory reservation"?

Reply:
@jinyunma I believe there should be some reference on VM's CPU and memory reservation. There are chances that user might get puzzled on how much the reservation should be.